src/main/index.ts

```diff
@@ -59,6 +59,9 @@ import { UserStore } from "../common/user-store";
 import { WeblinkStore } from "../common/weblink-store";
 import { ExtensionsStore } from "../extensions/extensions-store";
 import { FilesystemProvisionerStore } from "./extension-filesystem";
+import { SentryInit } from "../common/sentry";
+
+SentryInit();
```

Comment:
Should we be calling this only if the user's preference is to report errors?

Reply:
We can't, because Sentry need to be init (Sentry.init()) as soon as possible.

The preference is checked before the exception being transport to Sentry sever, see 'beforeSend' hook.

Reply:
Yes, I'm just wondering about unnecessary sentry init, like network traffic. I can envision users that might be concerned that when they have reporting off Lens is still trying to talk to the sentry endpoint, even if just once on Lens startup. 

Good job!

Reply:
Yeah, I see your point, thanks for reviewing, helps a lot.